### PR TITLE
97 quisby

### PR DESCRIPTION
**Pull request by sousinha97**
aws prices based on machine and os and region

### CODE REVIEW

**Review by dvalinrh — APPROVED** [review]

File: quisby/pricing/cloud_pricing.py [hunk 1]
```diff
@@ -117,40 +146,50 @@ def get_aws_instance_info(instance_name, region):
...
+    if "PriceList" in response:
+        try:
+            price_list = json.loads(response["PriceList"][0])
+
+            # Extract pricing information as needed
+            terms = price_list["terms"]["OnDemand"]
+            for _, term_info in terms.items():
+                for _, price_dimension in term_info["priceDimensions"].items():
+                    price_per_hour = price_dimension["pricePerUnit"]["USD"]
+                    print(
```
Comment (frival) [inline quisby/pricing/cloud_pricing.py:169]: Only a question: do we want to leave this what looks like debugging line in?  I'm trying to decide from just looking at it if we'll get a bunch of output that might mask actual issues just by volume.

Comment (sousinha97) [reply quisby/pricing/cloud_pricing.py:169]: Are you talking about line 169 ?The print statement ?

Comment (frival) [reply quisby/pricing/cloud_pricing.py:169]: Yes, that's where github *should* have left the comment... :)

File: quisby/pricing/cloud_pricing.py [hunk 2]
```diff
@@ -177,8 +216,14 @@ def get_cloud_cpu_count(instance_name, region, cloud_type):
 
 
 if __name__ == "__main__":
-    # region = "us-east"
     # print(get_azure_pricing("Standard_D32s_v3",region))
-    # print(get_aws_cpucount("i3en.24xlarge",region))
     # print(get_gcp_prices("n2-standard-16",region)
-    pass
+    region = "us-east-1"  # Replace with your desired AWS region
```
Comment (frival) [inline quisby/pricing/cloud_pricing.py:221]: I'd assume we can pass these in on the command line rather than editing the python file? </stupidquestion>

**Pull request merged**